server/src/main/resources/routes

```diff
@@ -3,7 +3,10 @@
 # ~~~~
 
 # API routes
--> /api/v1  de.frosner.broccoli.ApiV1Router
+-> /api/v1  de.frosner.broccoli.routes.ApiV1Router
+
+# Downloads
+-> /downloads/ de.frosner.broccoli.routes.DownloadsRouter
```

Comment:
I don't think that this is an API endpoint.  

Can I send it JSON? Does it return JSON?  Do we use it as an API endpoint?  I don't think so: We use it as a static endpoint for the webui, and it's not intended to serve structured data but rather human-readable "logs" whatever they might contain.

As for `/downloads/`: It needs to be in a separate URL prefix to allow proper 404s for non-existing logs, and I couldn't come up with a better prefix.  While I agree that it's not a "static file" it's not a "dynamic view" either: The log won't refresh, so in a way you're really "downloading" a snapshot of the log at this particular point of time.